game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -66,6 +67,11 @@ class Cell(id: Int, startPosition: Vector2 = new Vector2(0f, 0f)) {
     return if (dir.magnitude > 0) dir.normalize * value else new Vector2(0f,0f)
   }
 
+  def contains(pos: Vector2): Boolean = {
+    return position.distanceTo(pos) <= mass
```

Comment:
Personally I saw it as a different task (once we do multiplayer). We can do the `X% overlapping other cell to eat` logic later.